dashboard/tool/src/main/webapp/WEB-INF/templates/course_dashboard.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml"
+      xmlns:th="http://www.thymeleaf.org">
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+        <link media="all" href="/library/skin/tool_base.css" rel="stylesheet" type="text/css" />
+        <link href="/library/webjars/fullcalendar/4.3.1/core/main.min.css" rel="stylesheet" />
+        <script src="/library/js/headscripts.js"></script>
+        <script src="/library/webjars/fullcalendar/4.3.1/core/main.min.js"></script>
```

Comment:
Should fullcalendar be added to headscripts instead of an independent call?

Reply:
Not using fullcalendar now. I need to remove that.